### PR TITLE
Add keycap styling with sample text

### LINKED ISSUE #7: Keycap styling
From https://github.com/elementary/blog/issues/9

>Looks like it's just monospace for now

We should be able to copy over some styles from the main site.

### PR DESCRIPTION
**Pull request by dar5hak**
Fixes #7 

I looked at these lines from the main site:

https://github.com/elementary/website/blob/2cf01ae216ba8b3598bdfd6b0ac55545581ad071/_styles/docs.css#L118-L142

And made a few necessary changes. This is what the result looks like:

image

The font size is larger than that on the main site, but so is the rest of the text, so I chose to keep it this way.

Does this look right?

Also, what should I do about the hex codes? The colors used here aren't defined as variables yet.

### CODE REVIEW

File: _sass/_typography.scss [hunk 1]
```diff
@@ -126,3 +126,22 @@ aside {
   margin-bottom: 2em;
 }
 
+kbd {
+  background-color: #f0f0f0;
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
  background-color: rgba(200, 200, 200, 0.15);
```

File: _sass/_typography.scss [hunk 2]
```diff
@@ -126,3 +126,22 @@ aside {
...
+  border: 1px solid #bfbfbf;
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
  border: 1px solid rgba(0, 0, 0, 0.30);
```

File: _sass/_typography.scss [hunk 3]
```diff
@@ -126,3 +126,22 @@ aside {
...
+  border-radius: 3px;
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
  border-radius: 0.2em;
```

File: _sass/_typography.scss [hunk 4]
```diff
@@ -126,3 +126,22 @@ aside {
...
+  box-shadow:
+    inset 0 1px 0 0 #fff,
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
    inset 0 1px 0 0 rgba(255, 255, 255, 0.12),
```

File: _sass/_typography.scss [hunk 5]
```diff
@@ -126,3 +126,22 @@ aside {
...
+    inset 0 -2px 0 0 #d9d9d9;
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
    inset 0 -2px 0 0 rgba(0, 0, 0, 0.15);
```

File: _sass/_typography.scss [hunk 6]
```diff
@@ -126,3 +126,22 @@ aside {
...
+  display: inline-block;
+  font-family: "Roboto Mono", monospace;
+  font-size: 80%;
+  line-height: 1.5;
+  margin: 3px;
+  min-width: 22px;
+  padding: 2px 4px;
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
  padding: 0.125em 0.5em;
```

File: _sass/_typography.scss [hunk 7]
```diff
@@ -126,3 +126,22 @@ aside {
...
+  border: 1px solid #bfbfbf;
+  border-radius: 3px;
+  box-shadow:
+    inset 0 1px 0 0 #fff,
+    inset 0 -2px 0 0 #d9d9d9;
+  display: inline-block;
+  font-family: "Roboto Mono", monospace;
+  font-size: 80%;
+  line-height: 1.5;
+  margin: 3px;
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
  margin: 0 0.2em;
```

File: _sass/_typography.scss [hunk 8]
```diff
@@ -126,3 +126,22 @@ aside {
...
+  min-width: 22px;
```
Comment (cassidyjames) [inline _sass/_typography.scss]:
```suggestion
  min-width: 1em;
```

**Comment by dar5hak** [comment]
You used transparency to make it look great on dark backgrounds, instead of going for the media query. That's genius! 🤯

**Comment by dar5hak** [comment]
Okay this looks much better:

image

Also, I am aware of your work regarding dark style preference, but as of today, is there a way to test that on a

**Comment by dar5hak** [comment]
Okay, this looks much better:

image

And with a dark style:

Screenshot_20191001-094734

**Comment by dar5hak** [comment]
A general question: I am aware of your efforts to push a standardized dark style preference, but as of today, is there a way of testing dark web apps on a default Juno install? What I do currently is launch a server on my Juno machine, then open the page on an Android connected to the same network.

**Comment by cassidyjames** [comment]
@dar5hak Yes! You can run Firefox with `GTK_THEME=elementary:dark firefox` to force a dark style.

**Comment by dar5hak** [comment]
Oh that's nifty! Thank you! 😁

**Pull request merged**